activation/nipost_state_test.go

```diff
@@ -58,6 +58,16 @@ func TestReadCRC(t *testing.T) {
 			data:   []byte("spacemesh"),
 			crc:    []byte{0xC7, 0x11, 0x73, 0xE0, 0x53, 0xD8, 0x75, 0x0F},
 			errMsg: "wrong checksum 0xC71173E053D8750F, computed 0xC61072DF52D7740E",
+		}, {
+			name:   "file too short",
+			data:   []byte("123"),
+			crc:    []byte{},
+			errMsg: "too small",
+		}, {
+			name:   "file empty",
+			data:   []byte(""),
+			crc:    []byte{0xC6, 0x10, 0x72, 0xDF, 0x52, 0xD7, 0x74, 0x0E},
+			errMsg: "wrong checksum 0xC61072DF52D7740E, computed 0x0",
 		},
```

Comment:
I'm confused by this test. The CRC64 of an empty file should be `0x0000000000000000` and if the file only contains those bytes it will pass reading & CRC validation

Reply:
What is confusing? It's exactly as you said. CRC of nothing is 0. The test file contains a wrong CRC and no data hence it should fail with "wrong checksum 0xC61072DF52D7740E, computed 0x0" message.

Reply:
but isn't this just a repetition of the "badCRC" testcase?

Reply:
No, it's an edge case when the input is empty.